messages/en.json

```diff
@@ -61,6 +60,99 @@
     "readTime": "{count, plural, =0 {less than a minute} one {# minute} other {# minutes}} read",
     "views": "Views"
   },
+  "about": {
+    "whatIsHackerspace": "What is Hackerspace?",
+    "inforAboutHackerSpace": "<p1> Hackerspace NTNU is a student-driven project open to all students, regardless of study direction or Hackerspace membership. We offer a creative arena where students from different disciplines can get help realizing their ideas in an engaged and inclusive environment. Here, you will find new technology at your disposal, including drones, 3D printers, and Virtual Reality equipment. </p1> <p2> Whether you are a first-year student needing help with your first Arduino project or a fourth-year student wanting to create a 3D model of Trondheim, we can provide both equipment and expertise. We also regularly hold courses for both beginners and advanced students in many exciting subjects. </p2> <p3> Come by to see what we're doing and have a chat. </p3> You can find us on the second floor of the A-block in the Realfagbygget at NTNU Gløshaugen.",
+    "showMaps": "Show map!",
+    "FAQ": "Frequently Asked Questions",
+    "canIUseThe3dPriner?": "Can I use the 3D printer?",
+    "answerCanIUseThe3dPriner": "<p1> All students can use the 3D printers and the rest of the workshop, regardless of whether they are members of Hackerspace or not. We have a reservation system available for all students with FEIDE login from the website's navigation bar. </p1> <p2> Come by during opening hours and get help from one of our members.</p2>",
+    "canITryVRGames/Equipment?": "Can I try VR games/equipment?",
+    "answerCanITryVRGames/Equipment": "Our VR equipment can be borrowed and tried out by all students, regardless of Hackerspace membership. We have a dedicated VR room where you can come and play as long as a Hackerspace member is available for setup and teardown. The equipment can be borrowed through the usual inventory system. Come by during our opening hours if you're interested.",
+    "howDoIBecomeAMember?": "How do I become a member?",
+    "answerHowDoIBecomeAMember": "<p1> Hackerspace has ongoing admissions. You can apply whenever you want. We usually respond within 3 working weeks. </p1> <p2> Note that you do not need membership to borrow equipment or try something cool. Come by for a chat during our opening hours to learn more about our offerings and what it's like to be a member. </p2>",
+    "activeGroup": "Our Active Groups",
+
+    "example_title": "Lorem Ipsum",
+    "example_text": "Lorem ipsum dolor sit amet. Aut rerum illum ut voluptatem enim qui consequatur exercitationem nam molestias fugit est expedita voluptas. Ad quas reiciendis non voluptas recusandae qui optio ratione."
+  },
+  "leaderboard": {
```

Comment:
leaderboard.leaderboard is a weird key. something like leaderboard.title or leaderboard.label makes more sense